apps/cookbook/src/app/examples/icon-example/icon-example.component.scss

```diff
@@ -1,4 +1,4 @@
-@import './libs/designsystem/src/lib/scss/utils';
+@import '~@kirbydesign/core/src/scss/utils';
```

Comment:
That is true. Looking at my suggestion again I think we should perhaps just ignore it? I have trouble seeing exactly what value it would add. It would also make it even more rigid to make changes to configuration/packaging, which I do not think is desirable.